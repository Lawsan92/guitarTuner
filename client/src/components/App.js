import React from 'react';
import Tuner from './Tuner.js';

const App = () => {
  return (
    <div className='app'>
      <Tuner/>
    </div>
  )
}

export default App;