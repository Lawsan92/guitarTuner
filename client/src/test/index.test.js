/**
 * @jest-environment jsdom
 */

import React from 'react';
import App from '../components/App.js';
import renderer from 'react-test-renderer';

test('use jsdom in this test file', () => {
  const element = document.createElement('div');
  expect(element).not.toBeNull();
});

it('renders <App> correctly', () => {
  const tree = renderer
    .create(
      <App/>
     )
    .toJSON();
  expect(tree).toMatchSnapshot();
})