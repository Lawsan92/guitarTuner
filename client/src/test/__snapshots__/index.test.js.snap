// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders <App> correctly 1`] = `
<div
  className="app"
>
  <div
    className="tuner"
  >
    Tuner
  </div>
</div>
`;
